src/js/components/Notification/__tests__/Notification-test.tsx

```diff
@@ -218,16 +219,14 @@ describe('Notification', () => {
   });
 
   test('custom theme', () => {
-    const theme = {
+    const theme: ThemeType = {
       notification: {
         direction: 'row',
-        truncation: true,
         container: {
           pad: 'medium',
         },
         toast: {
           direction: 'column',
-          truncation: false,
```

Comment:
Can we keep the `truncation: true` and `truncation: false`?

Reply:
Yes we can but the necessary changes are need to be made in base.d.ts

Reply:
but base.js does not have any truncation property for notification or notification.toast

Reply:
Ah gotcha, looks like we don't document this prop either. In that case I'm fine removing `truncation` from this test